pyproject.toml

```diff
@@ -14,7 +14,7 @@ babel = "==2.*,>=2.0.0,>=2.5.1"
 backoff = "==1.*,>=1.0.0,>=1.4.3"
 "backports.tempfile" = ">=1.0rc1"
 beard = "==0.*,>=0.0.0,>=0.2.0"
-celery = {git = "https://github.com/michamos/celery.git", branch = "fix-redis-connection-result-consumer"}
+celery = "==4.4.6"
```

Comment:
why not
```suggestion
celery = "^4.4.6"
```

Reply:
On HEP we are pinning to specific version as we decided some time ago that all packages should be pinned manually to specific version. Don't we want to do the same thing on NEXT?

Reply:
I was not aware of this policy. I don't really see the point (as packages are locked by the `poetry.lock` file anyway), but if that's the policy that's ok for me.